services/src/test/scala/cromwell/services/metadata/impl/MetadataDatabaseAccessSpec.scala

```diff
@@ -38,9 +38,16 @@ class MetadataDatabaseAccessSpec extends FlatSpec with Matchers with ScalaFuture
 
   def testWith(configPath: String): Unit = {
     lazy val dataAccess = new MetadataDatabaseAccess with MetadataServicesStore {
-      override val metadataDatabaseInterface =
-        new MetadataSlickDatabase(ConfigFactory.load.getConfig(configPath))
+      override val metadataDatabaseInterface = {
+        val databaseConfig = ConfigFactory.load.getConfig(configPath)
+
+        // NOTE: EngineLiquibaseSettings **MUST** always run before the MetadataLiquibaseSettings
+        new EngineSlickDatabase(databaseConfig)
+          .initialized(EngineServicesStore.EngineLiquibaseSettings)
+
+        new MetadataSlickDatabase(databaseConfig)
           .initialized(MetadataServicesStore.MetadataLiquibaseSettings)
+      }
```

Comment:
gotta ask what triggered this realization 🙂

Reply:
Running `sbt alltests:test` with sbt 1.0.3 on my laptop triggered the same error travis was seeing. Looking in the resulting mysql changelog rows, I could see that the metadata changelog had been run _before_ the default/engine changelog.

Reply:
so does this mean we can upgrade sbt?

Reply:
Yep. I've tested with sbt 1.0.3 and LGTM.

During previous tests last weekend, I had to up IntelliJ's sbt memory from 768 (I used 1768M), and give my shell .sbtopts even more memory to run `sbt alltests:test` (I used 3072M), but I haven't retried recently since I also slimmed some of the redundant library dependencies.